src/style.css

```diff
@@ -1,3 +1,91 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+
+  --blue-color: #00acdc;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Regular.ttf);
+}
+
+.navigation {
+  display: flex;
+  width: 100vw;
+  justify-content: flex-end;
+  flex-direction: row;
+  position: relative;
+  margin-right: 30px;
```

Comment:
Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both). Here, you've added a right margin to `.navigation`. It's better to consistently use either top or bottom margins to avoid unexpected layout shifts due to collapsing margins.